diff --git a/Dockerfile b/Dockerfile
deleted file mode 100644
index 985dc1f..0000000
--- a/Dockerfile
+++ /dev/null
@@ -1,40 +0,0 @@
-ARG ROOT_CONTAINER=ubuntu:focal-20210416@sha256:86ac87f73641c920fb42cc9612d4fb57b5626b56ea2a19b894d0673fd5b4f2e9
-
-FROM $ROOT_CONTAINER
-
-USER root
-ENV JUPYTER_TOKEN=0000
-ENV JUPYTER_PASS=0000
-ENV JUPYTER_IP=0.0.0.0
-ENV JUPYTER_PORT=8888
-ENV JUPYTER_ENABLE_LAB=yes
-
-
-ENV DEBIAN_FRONTEND noninteractive
-RUN apt update -y && \
-    apt install -y --no-install-recommends \
-    tini \
-    wget \
-    ca-certificates \
-    sudo \
-    locales \
-    fonts-liberation \
-    run-one \
-    python3 python3-pip python3-dev && \
-    apt-get clean && rm -rf /var/lib/apt/lists/* && \
-    echo "en_US.UTF-8 UTF-8" > /etc/locale.gen && \
-    locale-gen && \
-    pip3 install --upgrade pip
-
-COPY requirements.txt requirements.txt
-RUN python3 -m pip install -r requirements.txt
-
-WORKDIR /work
-
-CMD jupyter notebook \
-    --ip=${JUPYTER_IP} \
-    --port=${JUPYTER_PORT} \
-    --NotebookApp.token='' \
-    --NotebookApp.password='' \
-    --allow-root --no-browser
-
diff --git a/docker-compose.yml b/docker-compose.yml
deleted file mode 100644
index 1f93e18..0000000
--- a/docker-compose.yml
+++ /dev/null
@@ -1,24 +0,0 @@
-version: '3.9'
-
-services:
-  ml-env-jupyter:
-    container_name: ml-env-jupyter
-    build: ./
-    ports: 
-        - "8889:8888"
-    volumes: 
-        - "./:/work"
-    environment:
-      - JUPYTER_TOKEN=0000
-      - JUPYTER_PASS=0000
-#  ml-env-jetbrains:
-#    container_name: ml-env-jetbrains
-#    image: registry.jetbrains.team/p/prj/containers/projector-pycharm-p:latest
-#    ports:
-#      - 8887:8887
-#    environment:
-#      - PROJECTOR_USER_NAME=work
-#    volumes:
-#      - "./:/home/work"
-
-  
\ No newline at end of file
diff --git a/model.pkl b/model.pkl
deleted file mode 100644
index d5c68da..0000000
--- a/model.pkl
+++ /dev/null
@@ -1 +0,0 @@
-asdffff
\ No newline at end of file
diff --git a/test.mds b/test.mds
deleted file mode 100644
index 1b701ec..0000000
--- a/test.mds
+++ /dev/null
@@ -1,3 +0,0 @@
-version https://git-lfs.github.com/spec/v1
-oid sha256:5e43f6b24c4ea888af8a7647a3fc2efdd4e42fdbc5091a2da3c3bab0bc84779e
-size 5
